src/map/battle.cpp

```diff
@@ -3632,7 +3632,8 @@ static void battle_calc_skill_base_damage(struct Damage* wd, struct block_list *
 				}
 #endif
 				if(sd->status.party_id && (skill=pc_checkskill(sd,TK_POWER)) > 0) {
-					if( (i = party_foreachsamemap(party_sub_count, sd, 0)) > 1 ) { // exclude the player himself [Inkfish]
+					i = max(party_foreachsamemap(party_sub_count, sd, 0) - 1, 0); // Bonus Attack multiplier based on party members count doesn't include own player [Tydus1] [Inkfish]
+					if( i > 0 ) { 
```

Comment:
I think this is overcomplicating things.
Just keep it the old way and reduce the variable i inside the if, because there it is always > 1 in the current code.